specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/preview/2024-12-09-preview/confidentialledger.json

```diff
@@ -0,0 +1,2196 @@
+            "schema": {
+              "$ref": "common.json#/definitions/ConfidentialLedgerError"
+            }
+          }
+        }
+      }
+    },
+    "/app/userDefinedFunctions/{functionId}": {
+      "delete": {
+        "operationId": "DeleteUserDefinedFunction",
+        "x-ms-examples": {
+          "DeleteUserDefinedFunction": {
+            "$ref": "./examples/ledger/DeleteUserDefinedFunction.json"
+          }
+        },
+        "summary": "Deletes a user defined function from the Confidential Ledger.",
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/UserDefinedFunctionIdParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "The specified user defined function is deleted."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "common.json#/definitions/ConfidentialLedgerError"
+            }
+          }
+        }
+      },
+      "get": {
+        "operationId": "GetUserDefinedFunction",
+        "x-ms-examples": {
+          "GetUserDefinedFunction": {
+            "$ref": "./examples/ledger/GetUserDefinedFunction.json"
+          }
+        },
+        "summary": "Gets a user defined function.",
+        "description": "Returns the user defined function in the Confidential Ledger",
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/UserDefinedFunctionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Gets the user defined function.",
+            "schema": {
+              "$ref": "#/definitions/UserDefinedFunction"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "common.json#/definitions/ConfidentialLedgerError"
+            }
+          }
+        }
+      },
+      "put": {
+        "operationId": "CreateUserDefinedFunction",
+        "x-ms-examples": {
+          "CreateUserDefinedFunction": {
+            "$ref": "./examples/ledger/CreateOrUpdateUserDefinedFunction.json"
+          }
+        },
+        "summary": "Creates a user defined function.",
+        "description": "Creates the user defined function in the Confidential Ledger",
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/UserDefinedFunctionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/UserDefinedFunctionParameter"
+          }
+        ],
+        "responses": {
+          "201": {
```

Comment:
Missing a 200 response (in case of update of existing function)

Reply:
Fixed, thanks